kujira/app/mirage/config.js

```diff
@@ -1,5 +1,49 @@
+import config from '../config/environment';
+
 export default function() {
 
+this.namespace = config.APP.NAMESPACE;
+
+this.get('/osds', function(db/*, request*/){
+  return {
+    data: db.osds.map(attrs => (
+      { type: 'osd', id: attrs.id, attributes: attrs}
```

Comment:
When you return list in JSON API, then type should be prural: [JSON API](http://jsonapi.org/format/#document-resource-objects)